docs/securities-lending.md

```diff
@@ -45,54 +45,50 @@ In a cash loan, the lender lends the borrower the requested securities,
 and the borrower provides cash as collateral against the securities that they 
 are borrowing. 
 
-The security that is being lent is modelled in the main trade `economicTerms` 
-using a single `AssetPayout`, which includes an `underlier` which identifies
-the `Security` explicitly within the data type `Instrument`.
-
-Loans against cash collateral are held under `economicTerms->collateral`
-with a `collateralPosition` containing one or more `product` data types, each of
-which is `TransferableProduct` which defines the asset being used as collateral
-(ie `Cash`) with the addition of `economicTerms` which define the rates and
-other terms payable using an `InterestRatePayout`.
+Loans against cash collateral are formed as a `SettlemenetPayout` that 

```

Comment:
This is incorrect, this page cannot be submitted. Lionel/Payal updated this in a previous PR and this will overwrite the corrections.